packages/fast-usdc/src/constants.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Status values for the StatusManager.
+ *
+ * @enum {(typeof TxStatus)[keyof typeof TxStatus]}
+ */
+export const TxStatus = /** @type {const} */ ({
+  /** tx was observed but not advanced */
+  Observed: 'OBSERVED',
+  /** IBC transfer is initiated */
+  Advanced: 'ADVANCED',
+});
+
+/**
+ * Status values for the VStorage Writer.
+ *
+ * @enum {(typeof WriterTxStatus)[keyof typeof WriterTxStatus]}
+ */
+export const WriterTxStatus = /** @type {const} */ ({
```

Comment:
helpful distinction. please make the unqualified name be this all-encompassing enum. `Observed` and `Advanced` are the subset of a transaction status that are recorded in local state.